decksite/data/archetype.py

```diff
@@ -35,8 +35,11 @@ def load_archetype(archetype: Union[int, str], season_id: int = None) -> Archety
         arch.decks = []
     return arch
 
-def load_archetypes(where: str = '1 = 1', merge: bool = False, season_id: int = None) -> List[Archetype]:
-    decks = deck.load_decks(where, season_id=season_id)
+def load_archetypes(where: str = '1 = 1', merge: bool = False, season_id: int = None, tournament_only: bool = False) -> List[Archetype]:
+    having = '1 = 1'
+    if tournament_only:
+        having = 'ct.name != "League"'
+    decks = deck.load_decks(where, season_id=season_id, having=having)
```

Comment:
Why does this need to be in the HAVING rather than in the WHERE? Each deck has exactly one competition_type so I think it's possible to get it into the WHERE clause even though the query uses a GROUP BY?

Reply:
You're right, I'll switch it to WHERE.